frontend/src/i18n/locales/en.yaml

```diff
@@ -385,6 +385,20 @@ manage:
       referencing-pages: Referencing pages
       referencing-pages-explanation: 'This video is referenced on the following pages:'
       no-referencing-pages: No pages reference this video.
+      delete:
+        title: Delete Video
+        confirm: Delete Video?
+        cannot-be-undone: This action <strong>cannot</strong> be undone!
+        failed: Deleting the video failed.
+        failed-maybe: Deleting the video possibly failed.
+        pending: Deletion is pending.
+        tooltip:
+          failed: >
+            The deletion was requested {{time}} ago but still has not happened, so it might have failed.
```

Comment:
Haven't tested yet but I think the `ago` should be removed here as `{{time}}` should already contain the "ago"?